parametric/test_headers_tracestate_dd.py

```diff
@@ -446,7 +449,7 @@ def test_headers_tracestate_dd_propagate_propagatedtags(test_agent, test_library
             val = tag[index:]
 
             assert key.startswith("_dd.p.")
-            assert "t." + key[6:] + ":" + val.replace("=", ":") in dd_items4
+            assert "t." + key[6:] + val.replace("=", ":") in dd_items4
```

Comment:
not knowing much about the headers logic, I assume this is fine?

Reply:
Yeah, the code above splits out `val` with the leading `=` still there, and then we converted that to `:` and added an extra `:` just for good measure, while we only want one ':'.